src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -1396,6 +1435,109 @@ private boolean twoVsOneDivergent(CaseData caseData) {
             && FULL_DEFENCE.equals(caseData.getClaimant1ClaimResponseTypeForSpec()));
     }
 
+    private CallbackResponse setApplicantResponseDeadlineV1(CallbackParams callbackParams) {
```

Comment:
this method is not called/used anywhere?

Reply:
might be a leftover after updating with master. I'll remove it